fastserde/avro-fastserde-tests-common/src/test/avro/logicalTypesTest1.avsc

```diff
@@ -0,0 +1,203 @@
+{
+  "type": "record",
+  "name": "FastSerdeLogicalTypesTest1",
+  "namespace": "com.linkedin.avro.fastserde.generated.avro",
+  "doc": "Used in tests to confirm fast-serde supports logical-types",
+  "fields": [
+    {
+      "name": "enumField",
+      "type": {
+        "name": "JustSimpleEnum",
+        "type": "enum",
+        "symbols": [
+          "E1",
+          "E2",
+          "E3",
+          "E4",
+          "E5"
+        ]
+      }
+    },
+    {
+      "name": "arrayOfEnums",
+      "type": [
+        "null",
+        {
+          "type": "array",
+          "items": "JustSimpleEnum"
+        }
+      ],
+      "default": null
+    },
+    {
+      "name": "mapOfEnums",
```

Comment:
As above.

Reply:
There is/was an issue when fast-generic-serializer encodes specific class with enums.
The fix has been extracted to separate PR:
- https://github.com/RTBHOUSE/avro-util/pull/4